packages/wct-mocha/README.md

```diff
@@ -0,0 +1,159 @@
+```
+
+## Run `wct`
+
+```bash
+$ wct --wct-package-name wct-mocha
+```
+
+## `.html` Suites
+
+Your test suites can be `.html` documents. For example,
+`test/awesomest-tests.html`:
+
+```html
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="../node_modules/mocha/mocha.js"></script>
+  <script src="../node_modules/chai/chai.js"></script>
+  <script src="../node_modules/wct-mocha/browser.js"></script>
+</head>
+<body>
+  <awesome-element id="fixture"></awesome-element>
+  <script type="module">
+    import {AwesomeElement} from '../src/awesome-element.js';
+    suite('<awesome-element>', function() {
+      test('is awesomest', function() {
+        const element = document.getElementById('fixture');
+        chai.assert.instanceOf(AwesomeElement);
+        chai.assert.isTrue(element.awesomest);
+      });
+    });
+  </script>
+</body>
+</html>
+```
+
+## `.js` Suites
+
+Alternatively, you can write tests in separate `.js` sources. For example,
+`test/awesome-tests.js`:
+
+```js
+suite('AwesomeLib', function() {
+  test('is awesome', function() {
+    assert.isTrue(AwesomeLib.awesome);
+  });
+});
+```
+
+## Special Features
+
+### Web Server
+
+If you prefer not to use WCT's command line tool, you can also run WCT tests
+directly in a browser via a web server of your choosing.
+
+Make sure that WCT's `browser.js` is accessible by your web server, and have
+your tests load `browser.js`.
+
+#### Nested Suites
+
+To help support this case, you can also directly define an index that will load
+any desired tests:
+
+```html
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <script src="../node_modules/mocha/mocha.js"></script>
+    <script src="../node_modules/wct-mocha/browser.js"></script>
+  </head>
+  <body>
+    <script>
+      WCT.loadSuites([
+        'awesome-tests.js',
+        'awesomest-tests.html',
+      ]);
+    </script>
+  </body>
+</html>
+```
+
+_When you use `wct` on the command line, it is generating an index like this for
+you based on the suites you ask it to load._
+
+
+
+# Nitty Gritty
+
+## Polymer
+
+By default, WCT will defer tests until `WebComponentsReady` has fired. This
+saves you from having to wait for elements to upgrade and all that yourself.
+
+If you need to test something that occurs before that event, the [`testImmediate` helper](https://github.com/Polymer/web-component-tester/blob/master/browser/environment/helpers.js#L29-41) can be used. Or, if you just want tests to run as soon as possible, you can disable the delay by setting `WCT.waitForFrameworks = false` (though, they are still async due to Mocha).
+
+
+## Mocha
+
+WCT supports Mocha's [TDD][mocha-tdd] (`suite`/`test`/etc) and [BDD][mocha-bdd]
+(`describe`/`it`/etc) interfaces, and will call `mocha.setup`/`mocha.run` for
+you. Just write your tests, and you're set.
+
+
+## Custom Environments
+
+If you would like to have WCT load libraries on your behalf, you can define a
```

Comment:
Why would anybody want to do this rather than just directly loading those scripts?

Reply:
I'll assume this feature came about because people wanted to conditionally load things from CLI based on environments, but I have no actual history available for why WCT provides this feature.  I can contrive reasons this exists, but that's all it would be.